### PR TITLE
Bump minor versions

### PR DESCRIPTION
**Pull request by joaopaulofonseca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,17 @@ A Bitcoin ABC docker image.
...
 ## Tags
 
-- `0.20.2-alpine`, `0.20-alpine`, `alpine` ([0.20/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.20/alpine/Dockerfile))
-- `0.19.5-alpine`, `0.19-alpine` ([0.19/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.19/alpine/Dockerfile))
-- `0.18.0-alpine`, `0.18-alpine` ([0.18/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.18/alpine/Dockerfile))
-- `0.17.1-alpine`, `0.17-alpine` ([0.17/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.17/alpine/Dockerfile))
+- `0.20.5-alpine`, `0.20-alpine`, `alpine` ([0.20/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.20/alpine/Dockerfile))
+- `0.19.12-alpine`, `0.19-alpine` ([0.19/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.19/alpine/Dockerfile))
+- `0.18.8-alpine`, `0.18-alpine` ([0.18/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.18/alpine/Dockerfile))
+- `0.17.2-alpine`, `0.17-alpine` ([0.17/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.17/alpine/Dockerfile))
```
Comment (adisney-up) [inline README.md]: Error building 0.17.2 on alpine:
`configure: error: libdb_cxx headers missing, Bitcoin ABC requires this library for wallet functionality (--disable-wallet to disable wallet functionality)`

File: README.md [hunk 2]
```diff
@@ -6,17 +6,17 @@ A Bitcoin ABC docker image.
 
 ## Tags
 
-- `0.20.2-alpine`, `0.20-alpine`, `alpine` ([0.20/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.20/alpine/Dockerfile))
-- `0.19.5-alpine`, `0.19-alpine` ([0.19/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.19/alpine/Dockerfile))
-- `0.18.0-alpine`, `0.18-alpine` ([0.18/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.18/alpine/Dockerfile))
-- `0.17.1-alpine`, `0.17-alpine` ([0.17/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.17/alpine/Dockerfile))
+- `0.20.5-alpine`, `0.20-alpine`, `alpine` ([0.20/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.20/alpine/Dockerfile))
```
[the remark this answers is not in the document]
Comment (joaopaulofonseca) [reply README.md]: Yes, you're right. I reverted it to the previous one, `0.20.4`.

File: 0.18/alpine/Dockerfile [hunk 3]
```diff
@@ -50,8 +50,8 @@ RUN set -ex \
     gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" ; \
   done
 
-ENV BITCOIN_ABC_VERSION=0.18.0
-ENV BITCOIN_ABC_SHASUM="11dc3ba7f193c70879b3fc3cc716fde56880dfebfab8bb556b7a355b2e64f09d  bitcoin-abc-${BITCOIN_ABC_VERSION}.tar.gz"
+ENV BITCOIN_ABC_VERSION=0.18.8
+ENV BITCOIN_ABC_SHASUM="9c3f21aec6fbd20bdaeb5667a96655dd6cbfb0ef66f6d26e0646c817f9fb1518  bitcoin-abc-${BITCOIN_ABC_VERSION}.tar.gz"
 ENV BITCOIN_ABC_PREFIX=/opt/bitcoin-abc-${BITCOIN_ABC_VERSION}
 
 RUN wget https://download.bitcoinabc.org/${BITCOIN_ABC_VERSION}/linux/bitcoin-abc-${BITCOIN_ABC_VERSION}.tar.gz
```
Comment (adisney-up) [inline 0.18/alpine/Dockerfile]:
```suggestion
RUN wget https://download.bitcoinabc.org/${BITCOIN_ABC_VERSION}/linux/src/bitcoin-abc-${BITCOIN_ABC_VERSION}.tar.gz
```

File: 0.19/alpine/Dockerfile [hunk 4]
```diff
@@ -50,8 +50,8 @@ RUN set -ex \
     gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" ; \
   done
 
-ENV BITCOIN_ABC_VERSION=0.19.5
-ENV BITCOIN_ABC_SHASUM="bdc13e2822cc4eba2c5079b1f0afc780d130292ee73ee2832eae2a793e770d88  bitcoin-abc-${BITCOIN_ABC_VERSION}.tar.gz"
+ENV BITCOIN_ABC_VERSION=0.19.12
+ENV BITCOIN_ABC_SHASUM="98df5eeb8e0f92b71fe02dd412fb5d542219ee9f5c8fefa4c42e076b8a309aa2  bitcoin-abc-${BITCOIN_ABC_VERSION}.tar.gz"
 ENV BITCOIN_ABC_PREFIX=/opt/bitcoin-abc-${BITCOIN_ABC_VERSION}
 
 RUN wget https://download.bitcoinabc.org/${BITCOIN_ABC_VERSION}/linux/src/bitcoin-abc-${BITCOIN_ABC_VERSION}.tar.gz
```
[the remark this answers is not in the document]
Comment (adisney-up) [reply 0.19/alpine/Dockerfile]: After compiling with this source, running the container fails with:
```
bitcoin-abc_1    | /entrypoint.sh: assuming arguments for bitcoind
bitcoin-abc_1    | /entrypoint.sh: setting data directory to /home/bitcoin/.bitcoin
bitcoin-abc_1    | 
bitcoin-abc_1    | su-exec: bitcoind: No such file or directory
```

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (joaopaulofonseca) [reply README.md]: I added back `0.20.5`, it's already available for download.

File: 0.19/alpine/Dockerfile (hunk 4 above)
[the remark this answers is not in the document]
Comment (joaopaulofonseca) [reply 0.19/alpine/Dockerfile]: Nice catch, probably I missed this one when testing the builds. I fixed it now.

File: README.md [hunk 5]
```diff
@@ -6,17 +6,17 @@ A Bitcoin ABC docker image.
...
+- `0.18.8-alpine`, `0.18-alpine` ([0.18/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.18/alpine/Dockerfile))
+- `0.17.2-alpine`, `0.17-alpine` ([0.17/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.17/alpine/Dockerfile))
 - `0.16.2-alpine`, `0.16-alpine` ([0.16/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.16/alpine/Dockerfile))
 - `0.15.1-alpine`, `0.15-alpine` ([0.15/alpine/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.15/alpine/Dockerfile))
 
-- `0.20.2`, `0.20`, `latest` ([0.20/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.18/Dockerfile))
-- `0.19.5`, `0.19` ([0.19/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.19/Dockerfile))
-- `0.18.0`, `0.18` ([0.18/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.18/Dockerfile))
-- `0.17.1`, `0.17` ([0.17/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.17/Dockerfile))
+- `0.20.5`, `0.20`, `latest` ([0.20/Dockerfile](https://github.com/uphold/docker-bitcoin-abc/blob/master/0.20/Dockerfile))
```
[the remark this answers is not in the document]
Comment (joaopaulofonseca) [reply README.md]: Hmm this one I'm not sure what is the issue. It builds just fine...

**Pull request closed without merging**